### PR TITLE
Updated circleci testing

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Updated circleci to test D8 and D9
- updated composer dependencies.

# Need Review By (Date)
- 1/15/2021

# Urgency
- medium

# Steps to Test
1. review passing tests

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -86,11 +86,11 @@
         "drupal/field_group": "^3.0",
         "drupal/layout_builder_restrictions": "^2.7",
         "drupal/menu_block": "^1.6",
-        "drupal/page_manager": "dev-4.x#78b492786c841c5d2eb07f6d977275a76b79e46f",
+        "drupal/page_manager": "^4.0",
         "drupal/paragraphs": "^1.11",
         "drupal/pathauto": "^1.8",
         "drupal/rabbit_hole": "^1.0-beta6",
-        "drupal/taxonomy_menu": "dev-3.x#c0892fa9b428ceb784876554c9b80f49d72a246f",
+        "drupal/taxonomy_menu": "dev-3.x",
```
Comment (sherakama) [inline composer.json:93]: If we're on the dev branch I'd rather see the hash with it as well.

File: stanford_news.install [hunk 2]
```diff
@@ -15,4 +15,6 @@ function stanford_news_install() {
   if ($tax_view = View::load('taxonomy_term')) {
     $tax_view->disable()->save();
   }
+
+  \Drupal::service('theme_installer')->install(['stable']);
```
Comment (sherakama) [inline stanford_news.install:19]: Why is this needed?

File: composer.json (hunk 1 above)
Comment (pookmish) [reply composer.json:93]: What if we did that in the profile instead of the module? easier dependency management

File: stanford_news.install (hunk 2 above)
Comment (pookmish) [reply stanford_news.install:19]: https://github.com/SU-SWS/stanford_news/blob/8.x-2.x/templates/field/field--node--su-news-dek--stanford-news.html.twig#L1

Comment (pookmish) [reply stanford_news.install:19]: the `standard` profile no longer installs the `stable` them OOTB. so we have to explicitly make sure its enabled. and unfortunately we can't do that via the info.yml file.

Comment (pookmish) [reply stanford_news.install:19]: Another option is to refactor the template and remove that `stable` dependency

Comment (sherakama) [reply stanford_news.install:19]: Hmm. I wonder if we can just drop the @stable namespace in the extend and go with simply `{% extends 'field.html.twig' %}`

Comment (sherakama) [reply stanford_news.install:19]: Perhaps try that instead.

Comment (pookmish) [reply stanford_news.install]: after looking at that template, it was completely unnecessary when a preprocess hook can add the classes just the same. I've removed the template and refactored it into a hook. I'd like to come back to the other templates in this module at some point. I feel like they were just put in to get by at the time and we can improve on them.

File: composer.json (hunk 1 above)
Comment (sherakama) [reply composer.json:93]: We should be good citizens and keep it locked here as we do share these modules across multiple projects and multiple profiles.

File: stanford_news.install (hunk 2 above)
Comment (sherakama) [reply stanford_news.install]: Sounds like a plan to me.
